projects/packages/masterbar/src/class-main.php

```diff
@@ -29,13 +29,13 @@ public static function init() {
 
 		new Admin_Color_Schemes();
 
-		if ( ( defined( 'WPCOM_ADMIN_BAR_UNIFICATION' ) && WPCOM_ADMIN_BAR_UNIFICATION ) || get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
+		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
 			return;
 		}
 
 		$host = new Host();
 
-		if ( ! $host->is_wpcom_simple() ) {
+		if ( ! $host->is_wpcom_simple() && ! ( defined( 'WPCOM_ADMIN_BAR_UNIFICATION' ) && WPCOM_ADMIN_BAR_UNIFICATION ) ) {
```

Comment:
I'm unsure but it seems to be okay to just disable Masterbar if `WPCOM_ADMIN_BAR_UNIFICATION` is enabled